extensions/oidc-client-reactive-filter/deployment/pom.xml

```diff
@@ -107,7 +107,7 @@
                         <configuration>
                             <skip>false</skip>
                             <systemPropertyVariables>
-                                <keycloak.docker.image>${keycloak.docker.legacy.image}</keycloak.docker.image>
+                                <keycloak.version>${keycloak.version}</keycloak.version>
```

Comment:
I don't think we can do it, we still have tests running against WildFly based distro

Reply:
why? isn't it much more desirable to test this module with Keycloak 25? anyway, I don't mind to drop it if you want. it's your area, just tell me where else can I test it.

Reply:
Yeah, if I recall it right it causes a legacy KC load, but with this change it would switch to the latest version. What dovuou need to test ?

Reply:
I changed this, because then this module uses Keycloak Test Server with 25 rather then legacy 19. It has only test `NamedOidcClientFilterDevModeTest` and it reproduces the issue. I pasted error message in the description of this PR, it is the error thrown by this test when last name and first name are not set. Therefore it is a reproducer.

I don't know what is policy, but I tried to look in the Keycloak.org and it seems that they strongly recommended to use Quarkus-based version https://www.keycloak.org/2023/03/adapter-deprecation-update.html (this is for adapters because it's the newest article I found), so I thought it is not bad change.

Reply:
Dev service for KC does not set those properties but all works. I think you may instead do exactly what it and 
some other tests do, load upconfig.json if it is non legacy

Reply:
Okay, I didn't realize that this `KeycloakTestResourceLifecycleManager` is not supposed to work with Keycloak 25. I'll read docs properly next time. You recommend there Dev Svcs, https://quarkus.io/guides/security-oidc-bearer-token-authentication#bearer-token-integration-testing-keycloak, though honestly I didn't get that I must use legacy.

I'll use Dev Svc.